text_generation/causal_lm/cpp/README.md

```diff
@@ -148,7 +148,6 @@ To enable Unicode characters for Windows cmd open `Region` settings from `Contro
    2. https://huggingface.co/microsoft/phi-1_5
 9. [notus-7b-v1](https://huggingface.co/argilla/notus-7b-v1)
 10. [zephyr-7b-beta](https://huggingface.co/HuggingFaceH4/zephyr-7b-beta)
-11. [redpajama-3b-chat](https://huggingface.co/ikala/redpajama-3b-chat)
```

Comment:
Why do you remove [redpajama-3b-chat](https://huggingface.co/ikala/redpajama-3b-chat)?